src/main/scala/org/broadinstitute/dsde/workbench/sam/directory/JndiDirectoryDAO.scala

```diff
@@ -28,6 +28,50 @@ class JndiDirectoryDAO(protected val directoryConfig: DirectoryConfig)(implicit
     val sn = "sn"
     val cn = "cn"
     val uid = "uid"
+    val groupUpdatedTimestamp = "groupUpdatedTimestamp"
+    val groupSynchronizedTimestamp = "groupSynchronizedTimestamp"
+  }
+
+  def init(): Future[Unit] = {
+    for {
+      _ <- removeWorkbenchGroupSchema()
+      _ <- createWorkbenchGroupSchema()
+    } yield ()
+  }
+
+  def removeWorkbenchGroupSchema(): Future[Unit] = withContext { ctx =>
+    val schema = ctx.getSchema("")
+
+    Try { schema.destroySubcontext("ClassDefinition/workbenchGroup") }
+    Try { schema.destroySubcontext("AttributeDefinition/" + Attr.groupSynchronizedTimestamp) }
+    Try { schema.destroySubcontext("AttributeDefinition/" + Attr.groupUpdatedTimestamp) }
+  }
+
+  def createWorkbenchGroupSchema(): Future[Unit] = withContext { ctx =>
+    val schema = ctx.getSchema("")
+
+    createAttributeDefinition(schema, "1.3.6.1.4.1.18060.0.4.3.2.200", Attr.groupUpdatedTimestamp, "time when group was updated", true, Option("generalizedTimeMatch"), Option("generalizedTimeOrderingMatch"), Option("1.3.6.1.4.1.1466.115.121.1.24"))
```

Comment:
These numbers are for their "global" directory thing that we're not really paying attention to right now, right?